test-case/volume-basic-test.sh

```diff
@@ -89,4 +89,7 @@ done
 #clean up background aplay
 pkill -9 aplay
 
+dlogi "Reset all PGA volume to 0dB"
+reset_PGA_volume || die "Failed to reset the PGA volume to 0dB."
+
```

Comment:
I know it's off-topic but please remove this spurious `exit 0` if you ever force-push a new version.